firestore-translate-text/functions/src/index.ts

```diff
@@ -98,6 +162,18 @@ const getChangeType = (
   return ChangeType.UPDATE;
 };
 
+const handleExistingDocument = async (
```

Comment:
We need to skip existing translations so users don't pay for duplicate translations.

Reply:
Done and tested: 
<img width="1190" alt="Screen Shot 2022-11-18 at 1 46 30 PM" src="https://user-images.githubusercontent.com/4635763/202808523-33428709-1726-44dc-8b23-d69b29866aa6.png">

Reply:
That makes sense. Anything that isn't backfill will be a change, and if the input changes, we need new translations for everything.